_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit Machines 244.
+
+## Shell: sudo is invoked only when explicitly requested
+
+Cockpit will no longer try to automatically run sudo on the very first login: you'll have to
+explicitly turn on administrative access once for each browser profile that you
+use to access Cockpit. This reduces a lot of noise about authentication failures in the journal.
+
+In addition, Cockpit now shows sudo's exact messages in the "Administrative access" dialog.
+
+![Sudo error messages](/images/sudo-error-messages.png)
+
+## Machines: Edit the MAC address of a VM's network interface
+
+To edit the MAC address of a virtual machine click the `Edit` button on the row
+of the network interface. A common scenario where this is needed is when
+configuring the DHCP server to map a static IP address to a certain MAC address.
+
+![VM edit MAC address](/images/vm-edit-mac-address.png)
+
+### Try it out
+
+Cockpit 244 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/244)
+ * [Cockpit Fedora 33](https://bodhi.fedoraproject.org/updates/FEDORA-2021-de41124d1b)
+ * [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/FEDORA-2021-16d3cfc061)
```

Comment:
As the announcement now speaks about c-machines as well, I think it should also mention the upstream/bodhi URLs for it?

Reply:
Done